.rhcicd/clowdapp-backend.yaml

```diff
@@ -121,12 +121,10 @@ objects:
               name: notifications-ephemeral-data
               key: ephemeral_data.json
               optional: true
-        - name: QUARKUS_REST_CLIENT_RBAC_AUTHENTICATION_READ_TIMEOUT
+        - name: RBAC_AUTHENTICATION_MP_REST_READTIMEOUT
           value: ${RBAC_AUTHENTICATION_READ_TIMEOUT}
         - name: INTERNAL_RBAC_ENABLED
           value: ${INTERNAL_RBAC_ENABLED}
-        - name: INTERNAL_ADMIN_ROLE
-          value: ${INTERNAL_RBAC_ADMIN}
```

Comment:
We still need this param over here

Reply:
Yea. I don't know where this came from.. will add it again